.github/workflows/increment-cargo-version-on-release.yml

```diff
@@ -0,0 +1,40 @@
+name: increment-cargo-version
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
```

Comment:
it make this action can be executed manually. seems you're using some github.event value in this action. not sure it still works fine if you trigger it directly